app/actions/callAPI.js

```diff
@@ -105,7 +105,7 @@ export default function callAPI({
     if (shouldUseCache) {
       const cachedRequest = getCachedRequest(state, endpoint, cacheSeconds);
       if (cachedRequest) {
-        return new Promise(() => dispatch(cachedRequest));
+        return new Promise(resolve => resolve(dispatch(cachedRequest)));
```

Comment:
isn't this the same as `return Promise.resolve(dispatch(cachedRequest))`?

Reply:
Yeah.. :smile: